src/main/java/blusunrize/immersiveengineering/common/data/Advancements.java

```diff
@@ -216,6 +216,16 @@ public void accept(Consumer<Advancement> consumer)
 					.withCriterion("railgun", InventoryChangeTrigger.Instance.forItems(Weapons.railgun))
 					.register(consumer, "immersiveengineering:main/craft_railgun");
 
+			ItemStack upgraded_railgun = new ItemStack(Weapons.railgun);
```

Comment:
_ in a non constant variable?

Reply:
Sorry, been writing a lot of python recently. Underscore naming is our variable pattern at work =P